### ISSUE TITLE
perform_reactome_analysis does'nt create a reactome visualization

### ISSUE THREAD

**Issue by SimonBedenbender** [reactome/ReactomeGSA#66]
Hello,

I tried to use this R package instead of the web application to analyze data with multiple samples which would simplify the annotation. I got it to work on expression data and sample data supplied as data.frames but the function never created a reactome visualization link.
The function _reactome_links_ doesnt yield any available links, the function _open_reactome_ yielded the error "Error in .local(x, ...) : Result does not contain any visualizations." indicating that no visualizations were created during the analysis. Also there was no information about reactome visualization creation in the verbose:

_Submitting request to Reactome API...
Compressing request data...
Reactome Analysis submitted succesfully
Converting dataset Reactome.0dvs7d...                                                                         
Mapping identifiers...                                                                                        
Performing gene set analysis using camera                                                                     
Analysing dataset 'Reactome.0dvs7d' using camera                                                              
Retrieving result..._ 

I also included disease pathways as it was stated in the information provided by get_reactome_methods that disabling it prevents the creation of reactome visualization.

Otherwise, I used the following code:

> Reactome.data.0dvs7d.Request <- ReactomeAnalysisRequest(method = "PADOG")
> 
> Reactome.data.0dvs7d.Request <- set_parameters(request = Reactome.data.0dvs7d.Request,
>                                                use_interactors = TRUE,
>                                                include_disease_pathways = TRUE,                                                                                      > > create_reactome_visualization = TRUE,
>                                                email = "simon.bedenbender@uni-marburg.de")
>                                             
> 
>    
> Reactome.data.0dvs7d.Request <- add_dataset(request = Reactome.data.0dvs7d.Request, 
>                                             expression_values = Reactome_data_0dvs7d,
>                                             sample_data = Reactome_0dvs7d_sample_data,
>                                             name = "Reactome.0dvs7d", 
>                                             type = "rnaseq_counts",
>                                             comparison_factor = "Timepoint", 
>                                             comparison_group_1 = "P0", 
>                                             comparison_group_2 = "P7"
>                                             )
> 
> Reactome.0dvs7d.result <- perform_reactome_analysis(request = Reactome.data.0dvs7d.Request, verbose = T)
> 
>  reactome_links(Reactome.0dvs7d.result)
> 
> open_reactome(Reactome.0dvs7d.result)

I could find any help in the normal R forums and in the ReactomeGSA manual. I would be very grateful, if someone could tell me what I am doing wrongly.

Many thanks in advance,

Simon

**Comment by jgriss** [comment]
Hi @SimonBedenbender,

Thanks a lot for your interest in ReactomeGSA!

I am very sorry that the tool did not work!

I failed to find any indication of an issue in our logs - but probably they were already deleted.

Could you please simply try again and if the error persists, post it here?

Thanks a lot!

Kind regards,
Johannes

**Comment by SimonBedenbender** [comment]
Hi Johannes,

thank you very much for your fast reply.

I think I somehow solved the issue. I am not 100% sure but I tried to import the data from a CSV file instead from a EXCEL file like I would do for the online tool. 
This somehow created a visualization. The creation wasn`t mentioned in the verbose, but _reactome_links_ contained the link and _open_reactome_ worked just wonderfully.

I noticed that RStudio replaced spaces in the column names with dots when using a CSV file but not when I used a EXCEL file. Also the _readxl_ function yield a tibble dataframe which I converted to a standard data.frame with _as.data.frame._ The _read.delim_ function produces a standard dataframe. Maybe this produced this strange behavior. 

Yesterday, I also started analysis using dataframes extracted from a Seurat Object (to differentially compare specific clusters whose cell type I already identified). This also worked without any issues with the visualization.

Thank you for your help!

Warm regards,
Simon
